src/components/App.tsx

```diff
@@ -2438,10 +2438,7 @@ class App extends React.Component<AppProps, AppState> {
       gesture.lastCenter = center;
 
       const distance = getDistance(Array.from(gesture.pointers.values()));
-      const scaleFactor =
-        this.state.elementType === "freedraw" && this.state.penMode
-          ? 1
-          : distance / gesture.initialDistance;
+      const scaleFactor = distance / gesture.initialDistance;
```

Comment:
did you revert his (in https://github.com/excalidraw/excalidraw/pull/4725/commits/f5fc2eb8a7a541257e1979518d4aa96f57a3f6c9) by mistake?

Reply:
mhm, judging from the PR title change, it was intentional. But allowing pinch-zooming in pen mode is causing issues while drawing again (for me, palm is regularly detected as a pinch zoom).

Reply:
Yes, I removed it intentionally... it did not cause issues for me, but if it is causing issues for you, I am sure It will for many others as well. So sadly I'll put this back. Having pinch zoom during freedraw is very valuable. you can zoom in to work on small detail, and zoom out for bigger strokes. Needing to change tools breaks the flow.

Reply:
> Having pinch zoom during freedraw is very valuable. you can zoom in to work on small detail, and zoom out for bigger strokes. Needing to change tools breaks the flow.

I agree. I was thinking there might be some heuristic to detect with high accuracy whether you're actually pinching with two fingers, but that's for another PR.